### PR TITLE
Colors update

### LINKED ISSUE #174: Provide more customization options to AutoSuggestBox
am using autosuggest box for search items.
I want to change background color properties of autosuggest box and the search icon should be in leading not trailing how can i achieve this?

### LINKED ISSUE #200: 🐛 FilledButton doesn't have Hover animation
See here on Input page (show the dialog): https://bdlukaa.github.io/fluent_ui/

### PR DESCRIPTION
**Pull request by bdlukaa**
See #111
Fixes #174
Fixes #200

## Changes

- Added `cardColor` and `menuColor` to `ThemeData`
- Implemented `Card`
- Exposed more properties to auto suggest box

## Pre-launch Checklist

- [x] I have run `dartfmt` on all changed files  
- [x] I have updated `CHANGELOG.md` with my changes  
- [ ] I have run "optimize/organize imports" on all changed files
- [ ] I have addressed all analyzer warnings as best I could
- [x] I have added/updated relevant documentation
- [ ] I have run `flutter pub publish --dry-run` and addressed any warnings

### CODE REVIEW

File: lib/src/controls/form/combo_box.dart [hunk 1]
```diff
@@ -309,12 +308,12 @@ class _ComboboxMenuState<T> extends State<_ComboboxMenu<T>> {
...
-                            (double total, double height) => total + height);
-                    final bool isScrollable =
-                        _kListPadding.vertical + menuTotalHeight >
-                            constraints.maxHeight;
+                    // final double menuTotalHeight = widget.route.itemHeights
+                    //     .reduce(
+                    //         (double total, double height) => total + height);
+                    // final bool isScrollable =
+                    //     _kListPadding.vertical + menuTotalHeight >
+                    //         constraints.maxHeight;
```
Comment (WinXaito) [inline lib/src/controls/form/combo_box.dart:316]: Delete these comment if not used ?

**Review by WinXaito — COMMENTED** [review]
With a light accent color, all widget are black
image

But with a darker accent color, there are differences between the widgets
image

**Comment by WinXaito** [comment]
With the dark infobar, we have a strange effect. (like he's come bigger) But it's probably because the color change is more important than light theme.
info_bar_dark

That we doesn't have with light theme
info_bar_light

**Comment by bdlukaa** [comment]
> But with a darker accent color, there are differences between the widgets

That's on purpose!

File: lib/src/controls/form/combo_box.dart (hunk 1 above)
Comment (bdlukaa) [reply lib/src/controls/form/combo_box.dart:316]: It'll be used in the future!

**Comment by WinXaito** [comment]
> > But with a darker accent color, there are differences between the widgets
> 
> That's on purpose!

Ok for the button, but does the checkbox be different than Switch ? (I didn’t check on Microsoft docs)

**Comment by bdlukaa** [comment]
@WinXaito for checkbox, the contrast color is applied

**Pull request merged**